media/js/firefox/new.js

```diff
@@ -27,6 +62,21 @@
                 // (from a download button) then we want them to see the same scene 2
                 // as non-firefox users and initiate a download
                 $html.addClass('firefox-latest');
+
+                uiTourGetConfiguration('appinfo', function(config) {
```

Comment:
I gave it a whirl, and while the code becomes a little less straight forward, it makes me *feel* better to not have a `document.ready` wrapped code block in a callback. Good suggestion. :+1: